src/components/ConnectedList.jsx

```diff
@@ -19,7 +20,17 @@ const ConnectedList = ({ t, connectors, children }) => (
         </button>
       </Link>
     </div>
-    <KonnectorList connectors={connectors} />
+    {connectors.length ? (
+      <KonnectorList connectors={connectors} />
+    ) : (
+      <div className="con-noKonnectors">
+        <img
+          src={noKonnectors}
+          className="con-noKonnectors--img"
```

Comment:
We are avoiding capital letters in class names, I suggest instead `con-no-konnectors--img`.

Reply:
Got it, thanks to @GoOz https://cozy.github.io/cozy-ui/styleguide/#naming-conventions